UmbracoFood.Core/Interfaces/IOrderRepository.cs

```diff
@@ -16,10 +16,12 @@ public interface IOrderRepository
 
         IEnumerable<Status> GetStatuses();
 
-        void AddOrderMeal(OrderedMeal orderedMeal);
+        void AddOrderedMeal(OrderedMeal orderedMeal);
 
         void ChangeStatus(int orderId, OrderStatus newStatus);
 
         void SetOrderIsInDelivery(int orderId, DateTime estimatedDeliveryTime);
+
+        void RemoveOrderedMeal(int orderedMealId);

```

Comment:
Dlaczego nie? Mniej możliwości kombinowania. Jak się to połączy w przyszłości z autoryzacją domenową, to będzie można usuwać tylko swoje pozycje na zamówieniu. Ewentualnie owner będzie mógł usuwać wszystko w danym zamówieniu